goto/commands/add.py

```diff
@@ -20,4 +20,4 @@ def add(magic, args):
     magic.add_shortcut(magicword, uri)
     magic.save()
 
-    return 'Added magic word %s' % magicword, None
+    return u'Added magic word %s' % magicword, None
```

Comment:
Is the prefix `u` a requirement on all strings?

Reply:
No.  
It was a misguided attempt at fixing unicode issues in python2, in py3  all string literals are unicode by default, and I now also imported `from __future__ import unicode_literals` to make it so for goto running on py2 also. 

The `u` is a slip. It can safely be removed